app/helpers/application_helper.rb

```diff
@@ -12,64 +12,15 @@ def body_id
     "#{controller.controller_name.dasherize}-#{controller.action_name.dasherize}"
   end
 
-  def nav_items
-    [
-      {
-        url: root_path,
-        controller: 'static_pages',
-        action: 'index',
-        title: 'Home'
-      },
-      {
-        url: ranking_index_path,
-        controller: 'ranking',
-        title: 'Ranking'
-      },
-      {
-        url: crags_path,
-        controller: 'crags',
-        title: 'Crags',
-      },
-      {
-        url: climbing_routes_path,
-        controller: 'climbing_routes',
-        title: 'Climbing routes',
-      },
-      {
-        url: rules_path,
-        controller: 'static_pages',
-        action: 'rules',
-        title: 'Rules'
-      },
-      {
-        url: contact_path,
-        controller: 'static_pages',
-        action: 'contact',
-        title: 'Contact'
-      },
-      {
-         url: about_path,
-         controller: 'static_pages',
-         action: 'about',
-         title: 'About'
-       }
-    ]
+  def navbar_helper
   end
 
-  def nav_helper style, tag_type
-    nav_links = ''
 
-    nav_items.each do |item|
-      nav_links << "<#{tag_type}><a href='#{item[:url]}' class='#{style} #{active?(item[:controller], item[:action])}'>#{item[:title]}</a></#{tag_type}>"
-    end
-    nav_links.html_safe
-  end
-
-  def active?(controller, action)
-    if controller_name == controller && action_name == action
-      "active"
-    elsif controller_name != 'static_pages' && controller_name == controller
-      "active"
+  def navbar_active(current_controller=nil, current_action=nil)
+    if current_controller && current_action
+      (controller_name ==current_controller && action_name == current_action) ? 'active' : ''
```

Comment:
I would suggest to return `active` if condition is `true`, because I see we don't care when it is `false`
```
return 'active' if (controller_name ==current_controller && action_name == current_action)
 ```